docs/basics/use_nwbwidgets.ipynb

```diff
@@ -53,7 +53,7 @@
    "metadata": {},
```

Comment:
<p>This could be reworded/restructured to seem a bit more natural with the rest of the text. Perhaps something like "In the cells below, the dandi api i used to facilitate the download. the method... etc. etc. and <code>get_asset_by_path</code> does blah"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>